pyproject.toml

```diff
@@ -1,11 +1,12 @@
 [tool.poetry]
-name = "cdf-tk"
+name = "cognite-toolkit"
 version = "0.1.0"
-description = "Official Cognite Data Fusion project templates"
-authors = ["Greger Wedel <greger.wedel@cognite.com>"]
+description = "Official Cognite Data Fusion tool for project templates and configuration deployment"
+authors = ["Cognite AS <support@cognite.com>"]
 license = "Apache-2"
 readme = "README.md"
 packages = [{include = "cdf_tk", from = "."}]
+include = ["cdf.py", "docs/**/**","common/**/*", "modules/**/*", "examples/**/*", "local_modules/**/*", "default.*.yaml", "config.yaml", "local.yaml", "packages.yaml", "README.md"]
```

Comment:
Do we really include everything into the package? I think we need to put it all in one folder, as if we do not, it can easily get mixed up with other installations.